tasks/main.yml

```diff
@@ -33,8 +33,13 @@
   notify:
     - Restart Rsyslog
 
+- debug: msg="Adding papertrail config at {{ papertrail_remote_syslog_config_paths[ansible_service_mgr].path }}"
+
 - name: Configure Papertrail service definition
-  template: src="remote_syslog.conf.j2"
-            dest="/etc/init/remote_syslog.conf"
+  template: src="remote_syslog.{{ ansible_service_mgr }}.conf.j2"
```

Comment:
I think that we want the `notify` from the task below to remain here so that a restart is triggered if the configuration file is changed.